### ISSUE TITLE
CommandButton: actions are called twice

### ISSUE THREAD

**Issue by stephanrauh** [TheCoder4eu/BootsFaces-OSP#259]
I noticed this in the tab demo. The action is called twice. Someone else reported a similiar issue concerning datatable. Might be a general problem.

**Comment by XaeroDegreaz** [comment]
I'm also seeing this problem.

However, when I have a `commandButton` using a standard `value` for a label like:

```xml
<b:commandButton
	value="UP"
	action="#{questionPage.changeOrder('up', question, assessmentGroup)}"
	ajax="true"
	update="@form"
	rendered="#{question.questionOrder ne 1}" />
```

It does not double trigger.

If I use one with a glyphicon:

```xml
<b:commandButton
	icon="arrow-up"
	action="#{questionPage.changeOrder('up', question, assessmentGroup)}"
	ajax="true"
	update="@form"
	rendered="#{question.questionOrder ne 1}" />
```

I can reliably get it to double-trigger when clicking on the icon but if i click on the button outside the icon, then it does not double-trigger.

**Comment by stephanrauh** [comment]
Interesting. I looked for the error at a different part of the source code. The tab demo is a non-AJAX commandButton without icon. I'll keep it in mind, anyways - you might be up to something.

**Comment by stephanrauh** [comment]
Non-AJAX commandButtons generate HTML code that calls `BsF.ajax.cb(this, event);` without adding `return false;`. That amount to calling an AJAX request, followed suite by a traditional non-AJAX request. Removing the AJAX call solves the problem. Thing is, I'm sure the code in `AJAXRenderer`, lines 183-214, serves a purpose, so simply removing it is no good.

**Comment by stephanrauh** [comment]
I've fixed the bug and uploaded a developer snapshot to Maven Central. 

@XaeroDegreaz The bug I've found has nothing to do with the glyphicon. Would you mind to test your code again to verify the bug has been solved? If not, please reopen the issue.

**Comment by saptarshibasu** [comment]
Yes. The bug is fixed. I was facing the same issue since last week. Just now I have tested with the new jar and have found it working. The issue was happenning with Chrome only.

**Comment by stephanrauh** [comment]
Great! Thanks for giving us feedback!
